docs/CHANGELOG.asciidoc

```diff
@@ -37,6 +37,11 @@
 
 == {es} version 7.6.0
 
+=== New Features
+
+* Add computation of tree SHAP feature importance values (SHapley Additive
+exPlanation) for classification and regression. (See {ml-pull}857[#857].)
```

Comment:
I think it would be better to stress the user facing functionality, feature importance, and de-emphasise the implementation detail.

Maybe something more like this:

* Add feature importance values to classification and regression results (using tree SHapley Additive
 exPlanation, or SHAP). (See {ml-pull}857[#857].)

Reply:
I adjusted the text as you suggested.